sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -126,12 +135,32 @@ export interface CreateKeyOptions {
    */
   expires?: Date;
   /**
-   * @member {msRest.RequestOptionsBase} [requestOptions] Options for this request
+   * @member {coreHttp.RequestOptionsBase} [requestOptions] Options for this request
    */
-  requestOptions?: msRest.RequestOptionsBase;
+  requestOptions?: coreHttp.RequestOptionsBase;
   keySize?: number;
 }
 
+/**
+ * @interface
+ * An interface representing the optional parameters that can be
+ * passed to beginDeleteKey
+ */
+export interface BeginDeleteKeyOptions {
+  /**
+   * @member {coreHttp.RequestOptionsBase} [requestOptions] Options for this request
+   */
+  requestOptions?: coreHttp.RequestOptionsBase;
+  /**
+   * @member {number} [intervalInMs] Time between each polling
+   */
+  intervalInMs?: number;
+  /**
+   * @member {string} [resumeFrom] A serialized poller, used to resume an existing operation
+   */
+  resumeFrom?: string;
```

Comment:
do we want to make this `string|DeleteKeyPoller` and handle checking typeof(thing) === 'string' to know if we have to serialize it for you? Or is that an unlikely use-case?

Reply:
I don't think we should do this.

Reply:
Can add it later.